docs/src/pages/components/placeholder/react.mdx

```diff
@@ -9,114 +16,126 @@ import { ComponentStyleDisplay } from '@/components/ComponentStyleDisplay';
+Import the Placeholder component.
 
-```jsx
-import { Placeholder, Flex } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+<Example>
+  <DefaultPlaceholderExample />
+  <ExampleCode>
+    ```jsx file=./examples/DefaultPlaceholderExample.tsx 
+    
+    ```
 
-<Flex direction="column">
-  <Placeholder />
-  <Placeholder />
-  <Placeholder />
-</Flex>
-);
-```
+  </ExampleCode>
+</Example>
 
-<Flex direction="column">
-  <Placeholder />
-  <Placeholder />
-  <Placeholder />
-</Flex>
+### isLoaded
 
-There are two ways you can use the Placeholder component:
+Use `isLoaded` prop to indicate whether or not the content is loaded.
 
-1. Pass your loading state to the `isLoaded` prop (suggested)
-2. Use conditional rendering (e.g., ternary operator)
+<Example>
+  <PlaceholderIsLoadedExample />
+  <ExampleCode>
+    ```jsx file=./examples/PlaceholderIsLoadedExample.tsx
 
-```jsx
-let isLoaded = false;
+    ```
 
-<Flex direction="column">
-  <Placeholder isLoaded={isLoaded}>First way</Placeholder>
-  {isLoaded ? 'Second way' : <Placeholder />}
-</Flex>
-);
-```
-
-<TwoWays />
+  </ExampleCode>
+</Example>
 
 ### Sizes
 
 Use the `size` prop to change the Placeholder size. Available options are `small`, `large`, and none (default).
 
-```jsx
-<Placeholder size="small" />
-<Placeholder />
-<Placeholder size="large" />
-```
+<Example>
+  <PlaceholderSizeExample />
+  <ExampleCode>
+    ```jsx file=./examples/PlaceholderSizeExample.tsx
+
+    ```
 
-<Flex direction="column">
-  <Placeholder size="small" />
-  <Placeholder />
-  <Placeholder size="large" />
-</Flex>
+  </ExampleCode>
+</Example>
 
-## CSS Styling
+## Customization
 
 ### Target classes
 
 <ComponentStyleDisplay componentName="Placeholder" />
 
-### Global styling
+### Theme
 
-To override styling on all Placeholder components, you can set the Amplify CSS variables or use the built-in `.amplify-placeholder` class.
+You can customize the appearance of all Placeholders in your application with a [Theme](/theming).
 
-(Example needed)
+<Example>
+  <PlaceholderThemeExample />
+  <ExampleCode>
+    ```jsx file=./examples/PlaceholderThemeExample.tsx
 
-To replace the Placeholder styling, unset it:
-
-```css
-.amplify-placeholder {
-  all: unset;
-  /* Add your styling here*/
-}
-```
-
-Set the starting and ending colors for the Placeholder animation using a custom CSS class with the Amplify CSS variables:
+    ```
 
-```css
-/* styles.css */
-.custom-placeholder {
-  --amplify-components-placeholder-start-color: purple;
-  --amplify-components-placeholder-end-color: blue;
-}
-```
+  </ExampleCode>
+</Example>
 
-```jsx
-import './styles.css';
+### Global styling
 
-<Placeholder className="custom-placeholder" />;
-```
+To override styling on all Placeholder components, you can set the Amplify CSS variables or use the built-in `.amplify-placeholder` class.
 
-<Flex>
-  <Placeholder className="custom-placeholder" />
-</Flex>
+<Example>
```

Comment:
same request as previous PR, change `:root` to `[data-amplify-theme]` and use a classname and css file for the example so that it is closer to what the example is actually trying to demonstrate (and will more likely be broken if what we are trying to demonstrate doesn't work).